frontend/src/containers/signup-page/signup.page.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { useForm } from "react-hook-form";
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import styles from './styles.module.scss';
+import { RegisterPayloadKey, validationUserSchema } from 'healthcare-shared'
+
+interface IRegisterPayload {
+  [RegisterPayloadKey.NAME]: string;
+  [RegisterPayloadKey.SURNAME]: string;
+  [RegisterPayloadKey.EMAIL]: string;
+  [RegisterPayloadKey.PASSWORD]: string;
+  [RegisterPayloadKey.RETYPE_PASSWORD]: string;
+  [RegisterPayloadKey.PHONE]: string;
+  [RegisterPayloadKey.IS_STAFF]: boolean;
+}
```

Comment:
We need it for all application, so it would be better to move it to the shared folder